src/main/java/jpa/domain/Line.java

```diff
@@ -0,0 +1,51 @@
+package jpa.domain;
+
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import org.springframework.data.annotation.CreatedDate;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+@Entity
+public class Line {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @CreationTimestamp
+    private LocalDateTime createdDate;
+
+    @UpdateTimestamp
+    private LocalDateTime modifiedDate;
+
+    private String color;
+
+    @Column(unique = true)
+    private String name;
+
+    public Line() {}
```

Comment:
```suggestion
    protected Line() {}
```
기본생성자를 public으로 두는 것보다 최대한 외부 호출을 막을 수 있으면서 jpa로 처리할 수 있도록 protected로 하는걸 추천합니다